test/actions/katello/host/update_system_purpose_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'katello_test_helper'
+
+module Katello::Host
+  class UpdateSystemPurposeTest < ActiveSupport::TestCase
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+    include FactoryBot::Syntax::Methods
+
+    before :all do
+      User.current = users(:admin)
+      @host = FactoryBot.create(:host, :with_subscription)
+    end
+
+    describe 'update system purpose' do
+      let(:action_class) { ::Actions::Katello::Host::UpdateSystemPurpose }
+
+      it 'plans' do
+        service_level = 'Standard'
+        purpose_role = 'Red Hat Enterprise Linux Server'
+        purpose_usage = 'Production'
+        purpose_addons = ['Test Addon']
+        action = create_action action_class
+
+        plan_action action, @host, service_level, purpose_role, purpose_usage, purpose_addons
+
+        assert_equal(@host.subscription_facet.service_level, 'Standard')
```

Comment:
for these assertions, why not reference the variables you defined above instead of 'Standard' in this case

Reply:
Updated